clang/test/Driver/cheri/cheri-cc1as.S

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: mips-registered-target
```

Comment:
This should really be under CodeGen and not Driver, but for now adding the registered-target REQUIRES makes sense.

Reply:
:+1: